### PR TITLE
3.18 release blogs

### PR DESCRIPTION
**Pull request by kghopson**
These are the remaining blogs to be published. There are a couple of issues: 

- On the **combo post blog**, I could not get the video to display correctly in preview even though the example I stole it from worked just fine. The video ID is correct. ???

- The **Android** and **Python** search page images are huge for some reason in the **custom search pages** blog

Open to any suggestions

### CODE REVIEW

File: blogposts/2020/3.18/combo-post.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: 'Small but mighty new features'
+author: 'Loïc Guychard, Adam Harvey, Eric Fritz'
+publishDate: 2020-07-20T00:00-07:00
+tags: ["blog"]
+slug: small-and-mighty-features
+heroImage: https://about.sourcegraph.com/sourcegraph-mark.png
+published: true
+---
+## Improved add / update repository flow for users trying private versions of Sourcegraph (#[11044](https://github.com/sourcegraph/sourcegraph/issues/11044))
```
Comment (christinaforney) [inline blogposts/2020/3.18/combo-post.md:10]: Title should be an h1

```suggestion
# Improved add / update repository flow for users trying private versions of Sourcegraph (#[11044](https://github.com/sourcegraph/sourcegraph/issues/11044))
```

File: blogposts/2020/3.18/custom-search-pages.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+title: 'Custom search pages on Sourcegraph Cloud'
+author: 'Farhan'
+publishDate: 2020-07-20T00:00-07:00
+tags: ["blog"]
+slug: custom-search-pages
+heroImage: https://about.sourcegraph.com/sourcegraph-mark.png
+published: true
+---
+## Custom search pages on Sourcegraph Cloud (#[11526](https://github.com/sourcegraph/sourcegraph/issues/11526))
```
Comment (christinaforney) [inline blogposts/2020/3.18/custom-search-pages.md:9]: Title should be an h1

```suggestion
# Custom search pages on Sourcegraph Cloud (#[11526](https://github.com/sourcegraph/sourcegraph/issues/11526))
```

File: blogposts/2020/3.18/sourcegraph-health-alerts.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+title: 'Getting notified about the health of Sourcegraph is even easier'
+author: 'Robert'
+publishDate: 2020-07-20T00:00-07:00
+tags: ["blog"]
+slug: sourcegraph-health-notification
+heroImage: https://about.sourcegraph.com/sourcegraph-mark.png
+published: true
+
+## Getting notified about the health of Sourcegraph is even easier
```
Comment (christinaforney) [inline blogposts/2020/3.18/sourcegraph-health-alerts.md:9]: Title should be an h1

```suggestion
# Getting notified about the health of Sourcegraph is even easier
```

File: blogposts/2020/3.18/combo-post.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+![Add repository flow](./images/add-repository-flow.png "Privacy feedback in Sourcegraph UI")
+
+## Campaigns now support GitLab (#[11586](https://github.com/sourcegraph/sourcegraph/issues/11586))
+Developer: [Adam Harvey](https://github.com/LawnGnome)
+
+You asked, we implemented. We’ve received a lot of feedback from developers using GitLab as their code hosts that campaign capabilities would be useful, so we added
+them in the Sourcegraph 3.18 release. This makes campaigns available to more organizations and helps make Sourcegraph more universal. See Adam’s introductory video below:
+
+[![Campaigns for Gitlab](https://img.youtube.com/vi/KatiVJ4D3H4/maxresdefault.jpg)](https://youtu.be/KatiVJ4D3H4)
```
Comment (christinaforney) [inline blogposts/2020/3.18/combo-post.md:26]: You should use the following embed code for adding videos (you will need to update the URLs below.

```
<p class="container">
  <div style="padding:56.25% 0 0 0;position:relative;">
    <iframe src="https://www.youtube.com/embed/{VIDEO_ID}" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
  </div>
  <p style="text-align: center"><a href="https://youtu.be/{VIDEO_ID}" target="_blank">View on YouTube</a></p>
</p>
```

Comment (christinaforney) [reply blogposts/2020/3.18/combo-post.md:26]: You can find this by looking at the blog post template in the handbook in raw format: https://raw.githubusercontent.com/sourcegraph/about/master/handbook/product/product_management/release_blog_post_template.md

**Comment by christinaforney** [comment]
> * On the **combo post blog**, I could not get the video to display correctly in preview even though the example I stole it from worked just fine. The video ID is correct. ???

See the updated snippet I added as suggestion (and the blog post template to have access to this). You should use the embed URL for the video.

> * The **Android** and **Python** search page images are huge for some reason in the **custom search pages** blog

You should resize the images before uploading them for two reasons: 1. size on the page 2. Bloat to the repository due to the image size (also see my comments above about not adding images to this repository).

File: blogposts/2020/3.18/custom-search-pages.md (hunk 2 above)
Comment (kghopson) [reply blogposts/2020/3.18/custom-search-pages.md:9]: Oh, yes Herzog did upload to YT for me! I forgot to take it out of the folder, I'll do that now. And I'll also look at the bucket thing

**Pull request merged**